pkg/app/mysql.go

```
@@ -0,0 +1,244 @@
+
+	podname, containername, err := mdb.GetPodAndContainerName(mdb.namespace)
+
+	// exec into the pod and create the test database, read password from secret
+	loginMysql := []string{"sh", "-c", "mysql -u root --password=" + mdb.password + ""}
+	stdout, stderr, err := kube.Exec(mdb.cli, mdb.namespace, podname, containername, loginMysql, nil)
+	format.Log(podname, containername, stdout)
+	format.Log(podname, containername, stderr)
+	if err != nil {
+		log.Fatalf("Error while connecting to mysql database %s", err.Error())
+		return err
+	}
+
+	return nil
+}
+
+func (mdb *MysqlDB) Insert(ctx context.Context, n int) error {
+	podname, containername, err := mdb.GetPodAndContainerName(mdb.namespace)
+
+	if err != nil {
+		log.Fatalf("Error while getting the deployment and container %s", err.Error())
+	}
+	log.Info("MysqlDB: Inserting some records in  mysql instance.")
+	insertRecordCMD := []string{"sh", "-c", "mysql -u root --password=" + mdb.password + " -e 'use testdb; INSERT INTO pets VALUES (\"Puffball\",\"Diane\",\"hamster\",\"f\",\"1999-03-30\",NULL); '"}
+	for i := 0; i < n; i++ {
+		stdout, stderr, err := kube.Exec(mdb.cli, mdb.namespace, podname, containername, insertRecordCMD, nil)
+		format.Log(podname, containername, stdout)
+		format.Log(podname, containername, stderr)
+		if err != nil {
+			log.Fatalf("Error while inserting values to table %s", err.Error())
+			return err
+		}
+	}
+	return nil
+}
+
+func (mdb *MysqlDB) Count(context.Context) (int, error) {
+	podname, containername, err := mdb.GetPodAndContainerName(mdb.namespace)
+	selectRowsCMD := []string{"sh", "-c", "mysql -u root --password=" + mdb.password + " -e 'use testdb; select count(*) from pets; '"}
+	stdout, stderr, err := kube.Exec(mdb.cli, mdb.namespace, podname, containername, selectRowsCMD, nil)
+	format.Log(podname, containername, stdout)
+	format.Log(podname, containername, stderr)
+	if err != nil {
+		log.Fatalf("Error while selecting rows from table %s:", err.Error())
```

Please return an error here